app/helpers/database_helper.rb

```diff
@@ -2,10 +2,18 @@
 
 module DatabaseHelper
   def with_read_replica(&block)
-    ApplicationRecord.connected_to(role: :reading, prevent_writes: true, &block)
+    if ENV['REPLICA_DB_NAME'] || ENV['REPLICA_DATABASE_URL']
```

Comment:
Could we have a helper for this, rather than replicating the condition?

Reply:
Did so in this file, but not in the other one, because it would need to be a class method rather than an instance method, and also because we likely don't want to import the related helpers in all models.

Reply:
Could it be a class method? `DatabaseHelper.replica_enabled?`?